### ISSUE TITLE
about Permutation Test 

### ISSUE THREAD

**Issue by zdlspace0528** [xuyiqing/fect#22]
Dear Pro Xu,
I have a question about your Stata command —fect—,When I do Permutation Test and get p value=0?
Does this pass Permutation Test?
Many thanks in advance.

**Comment by xuyiqing** [comment]
Yes, I saw it on the other page. To keep better track of things, could you
send me and Ziyi and email?

**Comment by zdlspace0528** [comment]
What can i conclude from permutation test?
